docker_xylem/service.py

```diff
@@ -213,14 +272,18 @@ def _route_request(self, request):
         method = self.requestRouter.get(request.path)
 
         if not method:
+            self.log.warn(
+                '{request.path} is not implemented by plugin.',
+                request=request
+            )
             return "Not Implemented"
```

Comment:
Docker's docs don't specify what response it expects if a plugin does not implement a certain feature.